letsencrypt/cli.py

```diff
@@ -1620,6 +1620,11 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
         "security", "--strict-permissions", action="store_true",
         help="Require that all configuration files are owned by the current "
              "user; only needed if your config is somewhere unsafe like /tmp/")
+    helpful.add(
+        "automation", "--allow-subset-of-names", dest="allow_subset_of_names",
+        action="store_true", default=False,
```

Comment:
Similarly, using `default=False` is unnecessary when using `action="store_true". The default value is automatically `False`.